cip/cip68/src/main/java/com/bloxbean/cardano/client/cip/cip68/CIP68TokenTemplate.java

```diff
@@ -26,41 +26,37 @@ public abstract class CIP68TokenTemplate<T extends CIP68TokenTemplate> extends N
     CIP68TokenTemplate(Map map, String assetName, int assetNameLabel) {
         super(map);
         this.assetNameLabel = assetNameLabel;
-        this.assetName = assetName;
+        this.name = assetName;
     }
 
     public int getAssetNameLabel() {
         return assetNameLabel;
     }
 
-    public String getHexAssetFullName() {
+    public String getAssetNameAsBytes() {
```

Comment:
@Kammerlo Let's just return byte[] . Also do we need to use name.getBytes(UTF-8) to convert to bytes to avoid any encoding issue?